index.rst

```diff
@@ -27,27 +27,54 @@ instructions please see the :ref:`setup guide <setup>`.
       git clone https://github.com/<your_username>/cpython
       cd cpython
 
-3. Build Python, on Unix and macOS use::
+3. Build Python, on Unix and macOS use:
```

Comment:
```suggestion
3. Build Python:
```

The tab now shows instructions for Unix, macOS, and Windows.

Reply:
This bit still needs doing.

Reply:
Weird, I know I accepted that suggestion. Updating...